### ISSUE TITLE
Hide docker logs from `we run`

### ISSUE THREAD

**Issue by cirocosta** [wedeploy/cli#86]
Hey,

maybe we shouldn't be exposing docker logs as docker is just the engine that we use under the hood for managing container execution. That could be hidden under `-v`.

Currently:

```
 static  we run
Pulling WeDeploy infrastructure docker image. Hold on.
latest: Pulling from wedeploy/local

(or the pull logs)

Digest: sha256:38ca42eb20d35b55b1dadf8ea802d6ebf8e97f3c2d8d86193b1ced4510da202e
Status: Image is up to date for wedeploy/local:latest
● Starting WeDeploy. 6s
```

At the same time i feel like it's very important to be explicit about  the state that the CLI is: now it's downloading images ... now it's bringing the infrastructure up ... now it's ready. Do  you known whether that's gatherable through docker remote api?  Wdyt?

**Comment by henvic** [comment]
tl;dr: docker run logs are already hidden.

We call "docker pull" immediately before calling "docker run" for exactly the reason you opened this issue: we don't want to show logs for "docker run". The docker logs you are seeing are for the "docker pull" command, as it is important for the user to see progress is being done on the background (instead of keeping the text ui stale). Nothing from "docker run" is piped to the user. Even if she uses the --verbose mode. I have tried it and it was confusing and made everything more complicated (given that we also create some other log messages as well). If the user really wants to see logs for the infrastructure, she needs to use docker logs directly.

**Comment by henvic** [comment]
Please notice as we are running the docker :latest image we will always try to pull. This is only the case for production versions when the user doesn't have the image yet.

Running from master, anytime:

```
$ we run
Pulling WeDeploy infrastructure docker image. Hold on.
latest: Pulling from wedeploy/local

Digest: sha256:a9fab3ed22f60cfca2683dd9b7a027fafffa903697aa6185ea8f2303ecbfb303
Status: Image is up to date for wedeploy/local:latest
● Starting WeDeploy.. 25s
You can now test your apps locally. Press Ctrl+C to shut it down when you are done.
```

Production version should show something like this, when the user already has the latest docker image:

```
$ we run
● Starting WeDeploy.. 25s
You can now test your apps locally. Press Ctrl+C to shut it down when you are done.
```
